src/components/TransactionsInterpreter/helpers/dateFormatted.ts

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
+
+export function dateFormatted(
```

Comment:
This name is value, what exactly is dateFormatted? maybe something like formatDateByUtc or something?